shell/platform/android/io/flutter/plugin/editing/ScribePlugin.java

```diff
@@ -19,18 +20,20 @@
  *
  * <p>The plugin handles requests for scribe sent by the {@link
  * io.flutter.embedding.engine.systemchannels.ScribeChannel}.
+ *
+ * <p>On API versions below 33, the plugin does nothing.
  */
 public class ScribePlugin implements ScribeChannel.ScribeMethodHandler {
 
-  private final ScribeChannel mScribeChannel;
-  private final InputMethodManager mInputMethodManager;
-  @NonNull private final View mView;
+  @NonNull private final ScribeChannel mScribeChannel;
+  @NonNull private final InputMethodManager mInputMethodManager;
+  @NonNull public View mView;
```

Comment:
I think you want this to be private or at least public and @visibleForTesting not public. 
Then add a setter for the view. My gut is that over time if the view changes then you want to be informed and be able to setup new callbacks or other events.